cbang will no longer compile for me

Since the last couple of updates, cbang won't compile on my Arch Linux:

``src/cbang/util/Regex.cpp: In member function ‘bool cb::Regex::match_or_search(bool, const std::string&, Match&) const’:
src/cbang/util/Regex.cpp:134:29: error: ‘__gnu_cxx::__alloc_traits<std::allocator<std::basic_string_view<char> >, std::basic_string_view<char> >::value_type’ {aka ‘class std::basic_string_view<char>’} has no member named ‘as_string’
  134 |       m.push_back(groups[i].as_string());
      |                             ^~~~~~~~~
scons: *** [build/cbang/util/Regex.o] Error 1
scons: building terminated because of errors.``

--- a/src/cbang/util/Regex.cpp
+++ b/src/cbang/util/Regex.cpp
@@ -37,97 +37,106 @@
 using namespace cb;
 using namespace std;
 
-struct Regex::private_t {
-    RE2 re;
 
-    private_t(const string &pattern, const RE2::Options &opts) : re(pattern, opts) {
-        if (!re.ok())
-            THROW("Failed to parse Regex: " << re.error());
-    }
+struct Regex::private_t {
+  RE2 re;
+
+  private_t(const string &pattern, const RE2::Options &opts) :
+    re(pattern, opts) {
+    if (!re.ok()) THROW("Failed to parse Regex: " << re.error());
+  }
 };
 
+
 Regex::Regex(const string &pattern, bool posix) {
-    RE2::Options opts;
+  RE2::Options opts;
 
-    opts.set_log_errors(false);
+  opts.set_log_errors(false);
 
-    if (posix) {
-        opts.set_posix_syntax(true);
-        opts.set_perl_classes(true);
-        opts.set_word_boundary(true);
-    }
+  if (posix) {
+    opts.set_posix_syntax(true);
+    opts.set_perl_classes(true);
+    opts.set_word_boundary(true);
+  }
 
-    pri = new private_t(pattern, opts);
+  pri = new private_t(pattern, opts);
 }
 
-string Regex::toString() const {
-    return pri->re.pattern();
-}
+
+string Regex::toString() const {return pri->re.pattern();}
+
 
 unsigned Regex::getGroupCount() const {
-    int n = pri->re.NumberOfCapturingGroups();
-    return n < 0 ? 0 : (unsigned)n;
+  int n = pri->re.NumberOfCapturingGroups();
+  return n < 0 ? 0 : (unsigned)n;
 }
+
 
 const map<string, int> &Regex::getGroupNameMap() const {
-    return pri->re.NamedCapturingGroups();
+  return pri->re.NamedCapturingGroups();
 }
+
 
 const map<int, string> &Regex::getGroupIndexMap() const {
-    return pri->re.CapturingGroupNames();
+  return pri->re.CapturingGroupNames();
 }
+
 
 bool Regex::match(const string &s) const {
-    return RE2::FullMatchN(s, pri->re, 0, 0);
+  return RE2::FullMatchN(s, pri->re, 0, 0);
 }
+
 
 bool Regex::match(const string &s, Match &m) const {
-    return match_or_search(true, s, m);
+  return match_or_search(true, s, m);
 }
+
 
 bool Regex::search(const string &s) const {
-    return RE2::PartialMatchN(s, pri->re, 0, 0);
+  return RE2::PartialMatchN(s, pri->re, 0, 0);
 }
+
 
 bool Regex::search(const string &s, Match &m) const {
-    return match_or_search(false, s, m);
+  return match_or_search(false, s, m);
 }
+
 
 string Regex::replace(const string &s, const string &r) const {
-    string result = s;
-    RE2::Replace(&result, pri->re, r);
-    return result;
+  string result = s;
+  RE2::Replace(&result, pri->re, r);
+  return result;
 }
 
-string Regex::escape(const string &s) {
-    return RE2::QuoteMeta(s);
-}
+
+string Regex::escape(const string &s) {return RE2::QuoteMeta(s);}
+
 
 bool Regex::match_or_search(bool match, const string &s, Match &m) const {
-    unsigned n = getGroupCount();
+  unsigned n = getGroupCount();
 
-    vector<RE2::Arg> args(n);
-    vector<RE2::Arg *> argPtrs(n);
-    vector<re2::StringPiece> groups(n);
+  vector<RE2::Arg>         args(n);
+  vector<RE2::Arg *>       argPtrs(n);
+  vector<re2::StringPiece> groups(n);
 
-    // Connect args
+  // Connect args
+  for (unsigned i = 0; i < n; i++) {
+    args[i]    = &groups[i];
+    argPtrs[i] = &args[i];
+  }
+
+  auto fn = match ? &RE2::FullMatchN : &RE2::PartialMatchN;
+  if ((*fn)(s, pri->re, argPtrs.data(), n)) {
+    m.clear();
+    m.offsets.clear();
+
     for (unsigned i = 0; i < n; i++) {
-        args[i] = &groups[i];
-        argPtrs[i] = &args[i];
+      m.push_back(string(groups[i].data(), groups[i].size()));
+      m.offsets.push_back(groups[i].data() - s.data());
     }
 
-    auto fn = match ? &RE2::FullMatchN : &RE2::PartialMatchN;
-    if ((*fn)(s, pri->re, argPtrs.data(), n)) {
-        m.clear();
-        m.offsets.clear();
+    return true;
+  }
 
-        for (unsigned i = 0; i < n; i++) {
-            m.push_back(string(groups[i].data(), groups[i].size()));
-            m.offsets.push_back(groups[i].data() - s.data());
-        }
-
-        return true;
-    }
-
-    return false;
+  return false;
 }

